cmd/neofs-cli/modules/control.go

```diff
@@ -206,6 +208,36 @@ func init() {
 	initControlRestoreShardCmd()
 }
 
+func verifyResponseControl(cmd *cobra.Command,
+	sigControl interface {
+		GetKey() []byte
+		GetSign() []byte
+	},
+	body interface {
+		StableMarshal([]byte) ([]byte, error)
+	},
+) {
+	if sigControl == nil {
+		exitOnErr(cmd, errors.New("missing response signature"))
+	}
+
+	bodyData, err := body.StableMarshal(nil)
+	exitOnErr(cmd, errf("marshal response body: %w", err))
+
+	// TODO(@cthulhu-rider): #1387 use Signature message from NeoFS API to avoid conversion
```

Comment:
This is not mentioned in the linked issue.

Reply:
Issue proposes placing all crypto things in single package which will cover this case among other things.

Reply:
So it is not about reusing `refs.Signature` in `*.proto` files for the control service?

Reply:
Yes, in order to follow current pattern used in other places of `neofs-node` repo, and this pattern is going to be changed within #1387.